### ISSUE TITLE
can't pass `pluginArgs `  params when StartReplication

### ISSUE THREAD

**Issue by jason-liew** [jackc/pglogrepl#7]
there is ```pluginArgs``` in [pgx before v4.0.0](https://github.com/jackc/pgx/blob/v3.6.2/replication.go#L409)
```
func (rc *ReplicationConn) StartReplication(slotName string, startLsn uint64, timeline int64, pluginArguments ...string) (err error) 
```
but pglogrepl lost it.

**Comment by mike-henders** [comment]
I'm looking at creating a PR to add a PluginArguments map[string]string to ReplicationOptions struct. It looks like a trivial change, although I'm relatively new to both golang and the PostgreSQL replication protocol so I'll probably make a mess of it. :)

**Comment by mike-henders** [comment]
It isn't obvious from the PostgreSQL documentation what the format of the arguments are in the START_REPLICATION command. From reading the `[source for pg_recvlogical](https://github.com/postgres/postgres/blob/master/src/bin/pg_basebackup/pg_recvlogical.c#L234-L257)` it appears the format is ( "name" 'value', ... ) where the option_name is a double-quoted identifier and the value is single quoted. When I tried double-quoting both of them and neither of them the server returned a RequestForQuery message and did not start streaming.

**Comment by jason-liew** [comment]
I struggled with single quoted identifiers all day, and finally created this [PR](https://github.com/jackc/pglogrepl/pull/8)

**Comment by jackc** [comment]
Merged in #8
